src/main/java/com/rarchives/ripme/ripper/rippers/E621Ripper.java

```diff
@@ -25,118 +27,141 @@
  * @author
  */
 public class E621Ripper extends AbstractHTMLRipper{
-	private static Pattern gidPattern=null;
-	private static Pattern gidPattern2=null;
-	private static Pattern gidPatternPool=null;
+public static final int POOL_IMAGES_PER_PAGE = 24;
 	
-	private DownloadThreadPool e621ThreadPool=new DownloadThreadPool("e621");
+	private DownloadThreadPool e621ThreadPool = new DownloadThreadPool("e621");
 	
 	public E621Ripper(URL url) throws IOException {
 		super(url);
 	}
-
+	
```

Comment:
Please remove trailing whitespace throughout (and whitespace on lines with no other content),